examples/Basic/gui_foreign_instrument.py

```diff
@@ -0,0 +1,99 @@
+"""
+This example demonstrates how to use an instrument from another library, here
+InstrumentKit, in the pymeasure graphical interface.
+It is a modification of the `gui_custom_inputs.py`: The `MainWindow` is the
+same, the `TestProcedure` was adjusted to use an instrumentKit instrument
+instead of a random number generator.
+
+Run the program by changing to the directory containing this file and calling:
+
+python gui_foreign_instrument.py
+"""
+
+import sys
+import tempfile
+from time import sleep
+
+from pymeasure.experiment import Procedure, IntegerParameter, FloatParameter
+from pymeasure.experiment import Results
+from pymeasure.display.Qt import QtGui, fromUi
+from pymeasure.display.windows import ManagedWindow
+
+# Import the InstrumentKit
+from instruments.thorlabs.pm100usb import PM100USB
```

Comment:
Does IK have a "fake instrument" feature one could use to test this even when not having the hardware around? I'm not familiar enough to know.

Reply:
I changed it to use a "Test-Adapter", such that it works, even without an actual device connected.

Further I adjusted it to be based on the most basic gui example.